File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/swift/F14.P3.swift
func triArea() -> Double {
// BEGIN PROBE F14.P3
struct Tri: Shape {
    var b: Double
    var h: Double
    func area() -> Double { 0.5 * b * h }
}
let s: Shape = Tri(b: 3.0, h: 4.0)
return s.area()
// END PROBE F14.P3
}

@main
struct Main {
    static func main() {
        print(triArea())
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F14.P3.kt
import shapes.Shape

fun triArea(): Double {
// BEGIN PROBE F14.P3
class Tri(val b: Double, val h: Double) : Shape {
    override fun area(): Double = 0.5 * b * h
}

val s: Shape = Tri(3.0, 4.0)
return s.area()
// END PROBE F14.P3
}

fun main() {
    println(triArea())
}
